stubs/cwcwidth.pyi

```diff
@@ -0,0 +1,4 @@
+# official typing seems to require a second arg
+def wcswidth(s: str, n=100000000): ...
+
+__version__: str
```

Comment:
`cwcwidth` should have type support. If it's buggy, I'd prefer to fix `cwcwidth`.

Reply:
For now I've included the second argument which was defaulting to None, I'll look into updating the types for cwcwidth to make the second argument optional.

Reply:
I think it just needs a release. It's fixed in master.